packages/cfd/src/Containers/cfd-password-manager-modal.tsx

```diff
@@ -317,7 +331,7 @@ const CFDPasswordManagerModal = ({
         return localize('Manage password');
     };
 
-    const onChangeActiveTabIndex = i => {
+    const onChangeActiveTabIndex = (i: number) => {
         setIndex(i);
     };
 
```

Comment:
also we can remove `password_type` in the line 345

Reply:
Done.